paper/src/main/java/com/denizenscript/denizen/paper/PaperModule.java

```diff
@@ -78,6 +78,7 @@ public static void init() {
             ScriptEvent.registerScriptEvent(PlayerTracksEntityScriptEvent.class);
         }
         ScriptEvent.registerScriptEvent(PlayerTradesWithMerchantScriptEvent.class);
+        ScriptEvent.registerScriptEvent(PlayerQuitsScriptEventPaperImpl.class);

```

Comment:
You're registering the paper impl here, but you haven't prevented registration of the spigot impl -- I'm not sure offhand exactly how this shows up in testing, but this should've caused some form of error in logs or double-firing when you test it